node_loaders/generate-social-img-loader.js

```diff
@@ -34,7 +34,7 @@ module.exports = function() {
         width: 1200,
         height: 630,
         subs: {
-          '#date': `${dateText} at 7 PM`,
+          '#date': `${dateText} at 7 PM Pacific`,
```

Comment:
Should this be shorter, like simply `Pac`, so it fits nicely into the left pane?

https://github.com/wafflejs/wafflejs.github.io/blob/master/images/social-2020-05.jpg

Reply:
is "PT" known well enough to mean Pacific standard or daylight savings?

Reply:
Oh yeah PT should be sufficient.